app/cmd/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+	"github.com/TudorHulban/GoLayouter/app/service"
+	"github.com/TudorHulban/GoLayouter/domain/objects"
+)
+
+func main() {
+	fileSource := os.Args[1]
+
+	content, err := helpers.ReadFile(fileSource)
+	if err != nil {
+		log.Print(err)
+	}
+
+	entries := objects.NewEntries(content).Parse()
+	serv := service.NewService(entries)
+
+	err = serv.WriteToDisk()
+	if err != nil {
+		log.Print(err)
```

Comment:
if there is an error we need to exit with an os exit different than zero

please use an os exit 2 
use fmt and not log also

Reply:
adddressed in d9c081de52e70f86c4ee0af1d15f3af69343fabf